x/marker/module.go

```diff
@@ -91,16 +92,19 @@ type AppModule struct {
 	accountKeeper  authkeeper.AccountKeeper
 	bankKeeper     bankkeeper.Keeper
 	feegrantKeeper feegrantkeeper.Keeper
+	govKeeper      govkeeper.Keeper
```

Comment:
I think you want `types.GovKeeper` here (and as an argument to `NewAppModule`. That way, this module (or at least this file) doesn't have a direct dependency on the gov module.